package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@reduxjs/toolkit": "^1.9.5",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
```

Comment:
- [ ] Based on the project requirements, `react-redux` package should be installed as the project dependencies. KIndly install this package. 

![image](https://github.com/ragangithub/bookstore-react/assets/84156714/40392b55-49de-48df-8945-b314e8fdaed3)